src/odemis/dataio/tiff.py

```diff
@@ -1456,12 +1463,19 @@ def _addImageElement(root, das, ifd, rois, fname=None, fuuid=None):
             ardata.attrib["FocusDistance"] = "%.15f" % globalMD[model.MD_AR_FOCUS_DISTANCE]
         if model.MD_AR_PARABOLA_F in globalMD:
             ardata.attrib["ParabolaF"] = "%.15f" % globalMD[model.MD_AR_PARABOLA_F]
+
+    # Store polarization analyzer data if any
+    if any(rd in globalMD for rd in [model.MD_ARPOL_POLARIZATION,
```

Comment:
Typically for this type of `for x in ...`, it's preferable to use a tuple instead of list. That's because tuple are slightly faster/lighter. Of course, it's an extremely minor optimisation. (No need to fix it here)